include/unified_teleop/pnt_stmp_modifiers.hpp

```diff
@@ -0,0 +1,217 @@
+#ifndef PNT_STMP_MODIFIERS_INCLUDE_GUARD_HPP
+#define PNT_STMP_MODIFIERS_INCLUDE_GUARD_HPP
+
+#include "rclcpp/rclcpp.hpp"
+#include "unified_teleop/controller.hpp"
+#include "geometry_msgs/msg/point_stamped.hpp"
+#include "sensor_msgs/msg/joy.hpp"
+#include <cmath>
+
+namespace rosnu
```

Comment:
Much of the implementation of this file belongs in a .cpp file not in the header file

Reply:
I thought since most functions are fairly straight forward, that I could keep it there (to also free up the src folder)